runtime/doc/lsp.txt

```diff
@@ -1649,13 +1651,14 @@ get({filter})                                       *vim.lsp.inlay_hint.get()*
         • {client_id} (`integer`)
         • {inlay_hint} (`lsp.InlayHint`)
 
-is_enabled({bufnr})                          *vim.lsp.inlay_hint.is_enabled()*
+is_enabled({bufnr}, {client_id})             *vim.lsp.inlay_hint.is_enabled()*
```

Comment:
if more filters are being added then it's time to make this a "kwargs" arg instead of adding more parameters.

probably `is_enabled({filter})` is best, though `is_enabled({bufnr}, {filter})` could be acceptable (and `bufnr=nil` would be "all/any buffer")

```suggestion
is_enabled({bufnr}, {filter})             *vim.lsp.inlay_hint.is_enabled()*
```